src/components/invite/useInvites.js

```diff
@@ -193,7 +192,7 @@ export const useInviteBonus = () => {
   return [collected, getCanCollect, collectInviteBounty]
 }
 
-export const useCollectBounty = () => {
+export const useCollectBounty = navigateTo => {
```

Comment:
let's pass a whole navigation object instead

Reply:
what is the difference between navigation.navigate or screenProps.navigateTo?

Reply:
passing down screenProps now